core/src/Cabana_LinkedCellList.hpp

```diff
@@ -214,9 +214,15 @@ class LinkedCellList
         // Allocate the binning data. Note that the permutation vector spans
         // only the length of begin-end;
         std::size_t ncell = totalBins();
-        Kokkos::View<int *, memory_space> counts( "counts", ncell );
-        OffsetView offsets( "offsets", ncell );
-        OffsetView permute( "permute", end - begin );
+        Kokkos::View<int *, memory_space> counts(
+            Kokkos::view_alloc( Kokkos::WithoutInitializing, "counts" ),
```

Comment:
`counts` is initialized here
https://github.com/ECP-copa/Cabana/blob/1da220ff3dbb71bb7d49fdd31eed3d606a155ac0/core/src/Cabana_LinkedCellList.hpp#L233